.github/workflows/compute.yml

```diff
@@ -0,0 +1,44 @@
+name: Delegated Compute
+
+on:
+  workflow_dispatch:
+    inputs:
+      stateId:
+        required: true
+      eventName:
+        required: true
+      eventPayload:
+        required: true
+      settings:
+        required: true
+      authToken:
+        required: true
+      ref:
+        required: true
+
+jobs:
+  run:
+    runs-on: ubuntu-latest
+    permissions: write-all
+
+    steps:
+      - uses: actions/checkout@v4
+        with:
+          ref: ${{ inputs.ref }}
+
+      - name: Echo inputs
+        run: echo ${{ inputs.settings }}
+
+      - name: Setup Node
+        uses: actions/setup-node@v4
+        with:
+          node-version: "20"
+
+      - name: Install Bun
+        run: npm install -g bun
+
+      - name: Calling action
+        uses: ./
+        env:
+          SUPABASE_URL: ${{ secrets.SUPABASE_URL }}
+          SUPABASE_KEY: ${{ secrets.SUPABASE_KEY }}
```

Comment:
I think @pavlovcik renamed it to `SUPABASE_ANON_KEY` across all the repos?

Reply:
In this case we need service key because we need write access to the DB.

Reply:
What about Cloudflare KV instead of Supabase? Seems way easier to setup for maintainers

Reply:
Cloudflare KV is meant to be used in Workers, it won't work in Github Actions